src/components/AssetsTable/index.jsx

```diff
@@ -370,13 +540,18 @@ export class AssetsTable extends React.Component {
     return (
       <Modal
         open={this.state.modalOpen}
-        title={`Delete ${this.state.assetToDelete.display_name}`}
+        title={(<WrappedMessage
+          message={messages.assetsTableDeleteObject}
+          values={{
+            displayName: this.state.assetToDelete.display_name,
+          }}
+        />)}
         body={this.renderModalBody()}
-        closeText="Cancel"
+        closeText={<PlainI18nString message={messages.assetsTableCancel} />}
```

Comment:
This cannot be a `FormattedMessage`, see https://github.com/edx/paragon/pull/138